Dockerfile.rosetta

```diff
@@ -0,0 +1,48 @@
+# syntax=docker/dockerfile:experimental
+
+FROM golang:1.17.5-bullseye as build
+RUN apt update && apt install ca-certificates git make -y
+WORKDIR axelar
+
+COPY ./rosetta/go.mod .
+COPY ./rosetta/go.sum .
+RUN go mod download
+
+COPY . .
```

Comment:
will this replace the go mod and sum files again ? does this work as we expect it to ? did you try building locally and were you able to verify rosetta support ?

Reply:
The sum was done manually using : go mod tidy

Reply:
For rosetta support I double check with @haiyizxx

Reply:
Regarding to the verify, good point. There is no good way to check atm, can only make sure we are using the correct rosetta/go.mod when build. 
I can spin up a container with the built image to verify later

Reply:
@talalashraf I modified to use only debian based commands (it is a requirement).
I can build the image locally and interact with axelard.
We can trigger an end to end deployment (dockerhub test creds are used in gh action file).